templates/_bundle_builder/Dockerfile

```diff
@@ -1,73 +1,151 @@
 FROM --platform=${TARGETPLATFORM} liferay/base:latest
 
+#
+#  Common Start
+#
+
 RUN mkdir /build
 
-# Common
 RUN apt-get update && \
+    \
```

Comment:
@Akos-Kreutz, I opened this pull request in VSCode and I realized that the tabulation character used in this new Dockerfile is the blank space instead of tab. Are you experiencing the same behavior on your end?

If so, can you please use the tab instead?

BChan will request it for us :smile:

Reply:
I missed this one, sure I will change it to tabulator.

Reply:
Thanks